adam/ontology/ontology.py

```diff
@@ -61,6 +62,9 @@ class Ontology:
     action_to_description: ImmutableDict[OntologyNode, ActionDescription] = attrib(
         converter=_to_immutabledict, default=immutabledict(), kw_only=True
     )
+    _node_to_relations: ImmutableDict[
```

Comment:
Would it be easier just to take a set of relations and then do the indexing within the `Ontology` class?

Reply:
I believe this has been resolved with the newer commits.